docs/src/index.md

```diff
@@ -63,6 +63,7 @@ Pages = [
     "examples/cohen_bergstresser.md",
     "examples/arbitrary_floattype.md",
     "examples/custom_solvers.md",
+    "examples/gross_pitaevskii_custom_potential.md",
```

Comment:
Since the file names are also the names of the websites, I'd suggest to keep the name in sync with the point of the example. So here the point is not GPE (that was before) it is the custom potential, so I'd just use `custom_potential.md`.